supabase/config.toml

```diff
@@ -76,7 +76,7 @@ enabled = true
 # in emails.
 site_url = "http://localhost:3000"
 # A list of *exact* URLs that auth providers are permitted to redirect to post authentication.
-additional_redirect_urls = ["http://localhost:3000/login/reset-password", "http://localhost:3000/**"]
+additional_redirect_urls = ["http://localhost:3000/login/reset-password", "http://localhost:3000/**", "https://**cool-creponne-3e1272.netlify.app/**", "https://cool-creponne-3e1272.netlify.app/login/reset-password"]
```

Comment:
no I don't think we want this here. This field is for `additional` urls because the base url is configured on the production project in supabase to be kept out of the development environment